### ISSUE TITLE
After select character, i got error

### ISSUE THREAD

**Issue by ninazu** [eezstreet/OpenD2#19]
`libD2Client.so: undefined symbol: _Z13D2Common_InitP18D2ModuleImportStrcP16D2GameConfigStrcP16OpenD2ConfigStrc`

**Comment by vadimbanev** [comment]
Same here...

**Comment by NorthernLight1** [comment]
What OS are you guys using? Is it 32 or 64 bit?

**Comment by vadimbanev** [comment]
64 bit linux, XUbuntu 18.04

**Comment by ninazu** [comment]
Mint 19.1 x64

Linux version 4.15.0-47-generic (buildd@lgw01-amd64-001) (gcc version 7.3.0 (Ubuntu 7.3.0-16ubuntu3)) #50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019

**Comment by typedefn** [comment]
I have made a pull request which fixes this issue #28

**Comment by sunneo** [comment]
Same here(branch master)
but it stuck at loading screen after I blindly input character name.

**Comment by smirkybg** [comment]
Indeed, I also get stuck at the character "Loading..." screen. It doesn't proceed with entering the actual game.
